CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+
+# Git Workflow and Development Process
+
+
+
+We assume that our contributors are familiar with Git concepts and Git commands. If anyone needs a good reference we recommend Atlassian Bitbucket tutorial (see the referencelinks provided below).
```

Comment:
You could add this file in the top ChronoLog directory and not in this subdirectory.

Reply:
I think this should not even be included into the repository. This is more for internal discussion. I would say let's put it into the shared google drive instead of anywhere here in the repo.

Reply:
CONTRIBUTING.md  is in the top ChronoLog  directory.